examples/tutorials/data/cta.py

```diff
@@ -1,58 +1,56 @@
 """
-CTA with Gammapy
-================
+CTAO with Gammapy
+=================
 
-Access and inspect CTA data and instrument response functions (IRFs) using Gammapy.
+Access and inspect CTAO data and instrument response functions (IRFs) using Gammapy.
 
 Introduction
 ------------
 
-The `Cherenkov Telescope Array
-(CTA) <https://www.cta-observatory.org/>`__ is the next generation
+The `Cherenkov Telescope Array Observatory (CTAO) <https://www.ctao.org/>`__ is the next generation
 ground-based observatory for gamma-ray astronomy. Gammapy is the core
-library for the Cherenkov Telescope Array (CTA) science tools
+library for the Cherenkov Telescope Array Observatory (CTAO) science tools
 (`2017ICRC…35..766D <https://ui.adsabs.harvard.edu/abs/2017ICRC...35..766D>`__
 and `CTAO Press
-Release <https://www.cta-observatory.org/ctao-adopts-the-gammapy-software-package-for-science-analysis/>`__).
+Release <https://www.ctao.org/news/ctao-adopts-the-gammapy-software-package-for-science-analysis/>`__).
 
-CTA will start taking data in the coming years. For now, to learn how to
-analyse CTA data and to use Gammapy, if you are a member of the CTA
+CTAO will start taking data in the coming years. For now, to learn how to
+analyse CTAO data and to use Gammapy, if you are a member of the CTAO
 consortium, you can use the simulated dataset from the CTA first data
-challenge which ran in 2017 and 2018.
-
--  https://forge.in2p3.fr/projects/data-challenge-1-dc-1/wiki (CTA
-   internal)
+challenge which ran in 2017 and 2018 (https://forge.in2p3.fr/projects/data-challenge-1-dc-1/wiki).
```

Comment:
This can be updated to the internal SDC which is already available to CTAO members